fedora/client/openidbaseclient.py

```diff
@@ -154,6 +160,20 @@ def __init__(self, base_url, login_url=None, useragent=None, debug=False,
 
         # python-requests session.  Holds onto cookies
         self._session = requests.session()
+
+        # Also hold on to retry logic.
+        # http://www.coglib.com/~icordasc/blog/2014/12/retries-in-requests.html
+        if retries is not None:
+            prefixes = ['http://', 'https://']
+            for prefix in prefixes:
+                self._session.mount(prefix, requests.adapters.HTTPAdapter(
+                    max_retries=Retry(
+                        total=retries,
+                        status_forcelist=[500],
+                        backoff_factor=retry_backoff_factor,
+                    ),
+                ))
+
```

Comment:
So the number of retries has to be passed when we instantiate the object, we can't change it afterward or via ``send_request()``, do I read this correctly?